libs/ojsonnet/Eval_jsonnet_subst.ml

```diff
@@ -36,6 +36,8 @@ module H = Eval_jsonnet_common
 (* Standard library *)
 (*****************************************************************************)
 
+(* FIXME: Check all those [Lazy.t] for possible concurrent access. *)
```

Comment:
github issue if it needs to be addressed?

Reply:
#127